src/helpers/indexing-helper.php

```diff
@@ -279,4 +279,25 @@ public function get_limited_filtered_unindexed_count( $limit ) {
 		 */
 		return \apply_filters( 'wpseo_indexing_get_limited_unindexed_count', $unindexed_count, $limit );
 	}
+
+	/**
+	 * Retriggers the indexing notification.
+	 *
+	 * @param string $reason The reason for which we need re-indexing.
+	 * 
+	 * @return void.
+	 */
+	public function retrigger_indexing_notification( $reason = Indexing_Reasons::REASON_IMPORT_COMPLETED ) {
+		// First, let's delete all count transients.
```

Comment:
I don't think this list is complete, it doesn't include the limited counts, right?

See https://github.com/Yoast/yoast-test-helper/pull/127/files

Reply:
Hm, yeah you're right, we need to clean up the limited ones, as well. On it!